views/js/runner/api/itemRunner.js

```diff
@@ -418,7 +418,13 @@ define([
                     responses = provider.getResponses.call(this);
                 }
                 return responses;
-           }
+           },
+
+            renderFeedbacks : function renderFeedbacks(feedbacks, itemSession, done){
```

Comment:
Please add a comment. It's not clear what to expect and how.
Also what's the difficulty to test this new method ?